featuretools/primitives/standard/api.py

```diff
@@ -2,5 +2,7 @@
 from .aggregation_primitives import *
 from .binary_transform import *
 from .cum_transform_feature import *
+from .date_time_transform_primitive import *
+from .lat_long_transform_primitive import *
```

Comment:
Can we name this file different?
```suggestion
from .datetime_transform_primitives import *
from .latlong_transform_primitives import *
```

Reply:
Of course, I'll implement those name changes today.